soc/xtensa/espressif_esp32/esp32s3/Kconfig.series

```diff
@@ -12,5 +12,17 @@ config SOC_SERIES_ESP32S3
 	select XIP if !MCUBOOT
 	select HAS_ESPRESSIF_HAL
 	select CPU_HAS_FPU
+
+config SOC_ESP32S3_PROCPU
+	bool "Application runs in ESP32S3 PROCPU (core 0)"
+	depends on SOC_SERIES_ESP32S3
+	help
+	  When this SOC is enabled, it will run application on PROCPU (core 0). It will automatically
+	  enable AMP support by building, flashing and loading APPCPU (core 1) image if exists.
```

Comment:
Just a question: outside of the IPC sample, automatic building for dual core is future work, right?

Reply:
Hi, yes. This PR allows having multi-image application using openAMP, which is a good advance. But we have more plans to go on with.

Reply:
Sounds good, thanks!